.github/CODEOWNERS

```diff
@@ -522,6 +522,8 @@ pep-0663.txt  @ethanfurman
 pep-0664.rst  @pablogsal
 pep-0665.rst  @brettcannon
 # pep-0666.txt
+pep-0668.rst  @geofft
```

Comment:
This needs to be the core devs/sponsors of this PEP (i.e. people who have write access to this repo).

Reply:
That'd be @dstufft and... that's it. Given that there's a core dev among the authors, we technically don't need a sponsor.